pixi.toml

```diff
@@ -8,14 +8,13 @@ platforms = ["win-64", "linux-64", "osx-64", "osx-arm64"]
 conda-recipe-v2-schema = { path = ".", editable = true }
 
 [dependencies]
-pydantic = ">=2.8.2,<3"
```

Comment:
Why remove this?

Reply:
as it is a pure python wheel that is published, I thought that it would be better if we keep using PyPI dependencies only and not get the conda's

Reply:
That is not true :)
Actually we only publish it on conda forge and not on pypi.

Reply:
ah, my mistake about stating that we publish pypi wheels , but it's still using `sdist` in recipe https://github.com/conda-forge/staged-recipes/pull/27080/files#diff-d508595b117e4a8f1f989ca80878ab9b832929b637603eb8f742e7d1fe69417fR9

which will be provided by pip https://github.com/conda-forge/staged-recipes/pull/27080/files#diff-d508595b117e4a8f1f989ca80878ab9b832929b637603eb8f742e7d1fe69417fR14

Reply:
Yes?

But the dependencies still come from conda-forge?

Reply:
sorry, but I don't get the point here of keeping both pydantic in pyproject.toml and pixi.toml ( by pixi having first-class support of pypi dependencies  )

Reply:
If we would drop it from pyproject.toml, we couldn't publish it to pypi if we wanted to.

Until we have a way in pixi to automatically install dependencies declared in pyproject.toml from conda, duplication is the best way as far as I can tell.

Reply:
why in this case to not keep it only in `pyproject.toml`, and use pydantic and any other dependencies from pypi ( which pixi does provides for us ) - this unlock our possibility to publish recipe-format as wheel on pypi.
we still need to duplicate pydantic in conda-forge recipe.yaml, so we have 3 places where we duplicate it ( instead of 2 )